swingset-extensions/usage-details/project-list.js

```diff
@@ -10,12 +10,13 @@
  */
 const PROJECT_LIST = [
   'hashicorp/hashicorp-www-next',
-  'hashicorp/hashiconf-next',
-  'hashicorp/cloud.hashicorp.com',
-  'hashicorp/learn',
+  'hashicorp/web/hashiconf',
```

Comment:
Not sure how to get the web repos access here. Something in sourcegraph?

<img width="1006" alt="CleanShot 2023-04-03 at 09 37 20@2x" src="https://user-images.githubusercontent.com/825855/229526447-3be5cc78-7d39-4351-b946-bb01e80b46e3.png">

Reply:
@alexcarpenter see `web-platform-packages` example below, I think this is what we would do:
```suggestion
  ['hashicorp/web', '/apps/hashiconf'],
```